README.md

```diff
@@ -33,6 +33,12 @@ git version 1.7.6
 hub version 2.2.3
 ```
 
+if you want to use the latest unstable version, use:
```

Comment:
An excellent feedback, thank you. I'll incorporate the language. 

PS: I considered phrasing it the same way as you've just proposed but wasn't really sure if that is true so thanks for clearing that up.